modules/local/coverage_tsv.nf

```diff
@@ -24,7 +24,6 @@ process COVERAGE_TSV {
     """
     cut -f 4 $mosdepth | sed -e 1i"${prefix}_cov" > mosdepth.tsv
     paste ${countbusogenes} mosdepth.tsv > ${prefix}_coverage.tsv
-
```

Comment:
Why are we deleting this extra line? Having a blank line before version code is pretty standard in modules.

Reply:
Some of the other modules don't have this line which is why I deleted it but I can add the line to all of them to keep it consistent